### PR TITLE
fix: [code-288]: fix tag icon

### PR DESCRIPTION
**Pull request by cjlee01**
### Summary
fix the size of the tag icon on the side nav of the gitness module

#### Screenshots
image

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by tan-nhu — APPROVED** [review]

File: src/modules/60-code/components/SideNav/SideNav.tsx [hunk 1]
```diff
@@ -106,7 +106,8 @@ export default function CODESideNav(): React.ReactElement {
               icon="code-tag"
               textProps={{
                 iconProps: {
-                  size: 16
+                  size: 18,
+                  className: css.tagIcon
```
Comment (ishantmehtaharness) [inline src/modules/60-code/components/SideNav/SideNav.tsx:110]: Looks Good @cjlee01

**Review by ishantmehtaharness — COMMENTED** [review]
Looks Good, Can you add a screenshot for reference. thanks

**Comment by ishantmehtaharness** [comment]
> Looks Good, Can you add a screenshot for reference. thanks

I see its added, thanks

**Review by ishantmehtaharness — APPROVED** [review]
looks good

**Review by tan-nhu — APPROVED** [review]

**Review by sahithikolichala — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**